sw/device/silicon_creator/rom_ext/doc/ownership.md

```diff
@@ -40,13 +40,60 @@ A chip may be in one of the following states:
-- `LockedNone` The chip doesn’t have an owner and is unable to accept a new owner.
-  This is a nearly-fatal error state and requires intervention by *silicon\_creator* to recover the chip.
+- `Recovery` The chip doesn’t have an owner and is unable to accept a new owner.
+  This is a nearly-fatal error state and requires intervention by *sku\_creator* to recover the chip.
 
 When the chip is in one of the `Unlocked` states, the ROM\_EXT will unlock Owner Page 1 allowing firmware to write a new owner configuration into that location.
 
 The ownership state is stored in the `boot_data` record.
 
+```mermaid
+flowchart LR
+subgraph container["<font size=6>Ownership State Transitions</font>"]
+    direction LR
+    start[LockedOwner] --> unlock{BootSvc::OwnershipUnlock}
+    unlock -->|mode: Any| any[UnlockedAny]
+    unlock -->|mode: Endorsed| endorsed[UnlockedEndorsed]
+    unlock -->|mode: Update| self[UnlockedSelf]
+    any --> activate{BootSvc::OwnershipActivate}
+    endorsed --> activate
+    self --> activate
+    activate --> done[LockedOwner]
+    style container fill:none,stroke:black
+end
+```
+
+### Owner Configuration
+
+The owner configuration (sometimes called "owner block") allows an owner to tailor the configuration of the chip to their use case.
+The configuration allows the owner to control the flash configuration, application verification keys, rescue mode configuration and miscellaneous configuration of the chip.
+
+- The flash configuration allows the owner to set access control (read, program, erase) and storage policy (ECC, scrambling, high endurance) of various regions and INFO pages of flash.
+- The owner may add a number of application verification keys that permit code execution.
+  The application key configuration includes an execution domain (e.g. `prod`, `dev` or `test`) and a key diversifier constant allowing the owner to control the derivation of sealing keys by the OpenTitan key manager.
+- The rescue configuration allows the owner to configure the opertion of the rescue mode.
+  The configuration allows protecting flash regions from erasure during rescue and designating the allowed set of rescue commands (an owner may want to disable certain rescue commands if they feel those commands represent an attack or denial of serivce vector in their use case).
+- The miscellaneous configuration contains configuration for features that aren't well categorized into one of the above configurations:
+  - Enable or disable SRAM execution.
+  - Reset the BL0 minimum security version.
+
+#### Update Mode
+
+There are several allowed update modes for the ownership configuration.
+These update modes are motivated by two scenarios:
+
+- The Unlock Key is a very powerful key and owners may want to attenuate the power of the unlock key to mitigate the risk of the key leaking.
+- An owner may want to provide an update mechanism for devices in the field that does not require the unlock and activate procedure.
```

Comment:
If I understand correctly, the main reason an owner might want this is that it would allow configuration updates without a device reset; it might be worth stating that to help people understand why owners would want to do this instead of a locked update.

Reply:
A reset is always required to update the owner config.  What the owner may want to avoid is the nonce-and-boot-services dance of unlock/update config/activate.

Full procedure:
1. Get nonce, sign unlock request (phone home), write to boot-svc.
2. Reset
3. Get updated owner config (phone home), write to owner page 1.
4. Reset
5. Get nonce, sign activate request (phone home), write to boot-svc.
6. Reset
7. Confirm operating under new config; back to operational.

Abbreviated procedure:
1. Get updated owner config (phone home, or included in a fw update).
2. Reset
3. Confirm operating under new config; back to operational.

Reply:
Ah, I see, so it's more about replacing several round-trips and three resets with 0-1 round-trips and one reset? I initially misunderstood this, makes sense now.